examples/auto/auto.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"net/http"
+	"os"
+	"os/exec"
+	"os/signal"
+	"time"
+
+	"github.com/StreamSpace/ss-light-client/scp/engine"
+	"github.com/olivere/elastic"
+	"github.com/teris-io/shortid"
+
+	"github.com/StreamSpace/ss-light-client/examples/auto/objects"
+)
+
+const (
+	elasticIndexName = "user_documents"
+	elasticTypeName  = "user_document"
+)
+
+type Document struct {
+	ID        string          `json:"id"`
+	CreatedAt time.Time       `json:"created_at"`
+	Content   interface{}     `json:"content"`
+	Object    objects.FileObj `json:"object"`
+}
+
+type StatOut struct {
+	ConnectedPeers []string            `json:"connected_peers"`
+	Ledgers        []*engine.SSReceipt `json:"ledgers"`
+	Duration       time.Duration       `json:"duration"`
+}
+
+func main() {
+	jsonfile, err := os.Open("/home/env.json")
+	if err != nil {
+		log.Fatalf("Unable to open json file %s", err.Error())
+	}
+	url, err := ioutil.ReadAll(jsonfile)
+	if err != nil {
+		log.Fatalf("Unable to read data from json file %s", err.Error())
+	}
+	var str map[string]string
+	err = json.Unmarshal(url, &str)
+	if err != nil {
+		log.Fatalf("Failed to Unmarshal url %s", err.Error())
+	}
+	res, err := http.Get(str["customer_files"])
+	if err != nil {
+		log.Fatal("Customer api failed ", err.Error())
+	}
+	data, _ := ioutil.ReadAll(res.Body)
+	res.Body.Close()
+
+	sharedObjects := []*objects.FileObj{}
+
+	err = json.Unmarshal(data, &sharedObjects)
+	//if err != nil {
```

Comment:
Why is this error ignored?

If there is error here we will hit panic later when we try to access sharedObjects[]